gnome/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+find_package(PNG)
 if(WITH_EXAMPLES AND TARGET PNG::PNG)
```

Comment:
Hm, I thought the previous `find_package(PNG)` in the examples directory would be sufficient...

Maybe it's best (i.e. clearer) if we change it to just `if(TARGET heif-thumbnailer)` instead? Is that one define in this scope?

Reply:
I don't know much about cmake, but would assume only stuff defined in parent folders is available in subfolders. The `examples` is a sibling of `gnome` so things defined there are probably not available here?

Reply:
For variables, I think that is the case, but I was under the impression the targets were known globally wherever they were defined...

Reply:
Would you mind trying it out anyway?

Reply:
How about `if (TARGET heif-thumbnailer)` to test directly if the binary exists than testing it indirectly?

Reply:
This works :+1: 
Interesting though that the `TARGET PNG::PNG` does not work - or are external targets handled differently?

Reply:
No problem, thanks for your various contributions!